python/pyiceberg/table/sorting.py

```diff
@@ -112,20 +115,18 @@ class SortOrder(IcebergBaseModel):
     The order of the sort fields within the list defines the order in which the sort is applied to the data.
 
     Args:
-      order_id (int): The id of the sort-order. To keep track of historical sorting
+      order_id (int): An unique id of the sort-order of a table.
```

Comment:
I don't think we need "of a table" -- that assumes the context that uses the sort order.